python/using_pyairtable/requirements.txt

```
@@ -1,3 +1,2 @@
 python-dotenv
-pyairtable
-requests
\ No newline at end of file
+pyairtable
```

v1.5.0 introduced upsert in pyairtable so we should specify users need at least that version 

```suggestion
pyairtable>=1.5.0
```